chain/chaintest/action_test_helpers.go

```diff
@@ -73,8 +73,8 @@ func (test *ActionTest) Run(ctx context.Context, t *testing.T) {
 
 		output, err := test.Action.Execute(ctx, test.Rules, test.State, test.Timestamp, test.Actor, test.ActionID)
 
-		require.ErrorIs(err, test.ExpectedErr)
-		require.Equal(output, test.ExpectedOutputs)
+		require.ErrorIs(test.ExpectedErr, err)
```

Comment:
```suggestion
		require.ErrorIs(err, test.ExpectedErr)
```

From the spec for `require.ErrorIs()`:

> `func ErrorIs(t TestingT, err error, target error, msgAndArgs ...interface{})`
`ErrorIs` asserts that at least one of the errors in err's chain matches target. This is a wrapper for errors.Is.